web/src/containers/activity/EntryDetails.js

```diff
@@ -79,13 +79,13 @@ const EntryDetails = ({ activityIndex, fundingSource, activityKey, name }) => {
       >
         <Review heading={title} headingLevel={4} editContent={editContent} />
         <div className={collapsed ? 'visibility--print' : ''}>
-          <Overview activityIndex={activityIndex} />
-          <Goals aKey={activityKey} />
-          <Schedule aKey={activityKey} />
-          <Costs aKey={activityKey} />
-          <ContractorResources activityIndex={activityIndex} />
-          <CostAllocate aKey={activityKey} />
-          <StandardsAndConditions aKey={activityKey} />
+          <Overview activityIndex={index} />
```

Comment:
The prop into `EntryDetails` is called `activityIndex`. We can change that to `index` if that makes more sense, but in any case it has to be the same on both sides. 😛

```suggestion
          <Overview activityIndex={activityIndex} />
```

Reply:
I... don't remember touching these? 😳 I'm not entirely sure what happened here, but thank you for catching it and fixing it!